website/blog/2023-10-26-release-1.5.md

```diff
@@ -0,0 +1,219 @@
+- **Extension API improvements**: A boatload of improvements to the extension API enabling more goodness from Podman Desktop's extensions
+
+Podman Desktop 1.5 is now available. [Click here to download it](/downloads)!
+
+**CUTE SELKIES WILL GO HERE JUST YOU WAIT**
+
+<!-- EXAMPLE -->
+<!-- ![Podman-desktop-1-1-hero](img/podman-desktop-release-1.1/podman-desktop-release-1.1.png) -->
+
+---
+
+## Release Details
+
+<!-- DESCRIBE MAJOR CHANGES, INCLUDE IMAGES / VIDEO IF APPLICABLE -->
+
+<!-- EXAMPLE -->
+<!-- ### Podman v4.5.1 -->
+<!-- Podman Desktop 1.1 moves up to [Podman 4.5.1](https://github.com/containers/podman/releases/tag/v4.5.1). -->
+
+<!-- WHEN DESCRIBING NAVBAR SECTIONS, INCLUDE ICON -->
+<!-- EXAMPLE -->
+<!-- We've also added options in **<icon icon="fa-solid fa-cog" size="lg" />Settings > Preferences** to automatically check for and install extension updates. -->
+
+### Onboarding
+
+**Mo note: I have to fill this in more**
+
+- Notification registry for notifications displayed on the dashboard [#4333](https://github.com/containers/podman-desktop/pull/4333)
+
+### Command Palette
+
+A new, search-driven command palette is now available to enable quick access to various commands available across Podman Desktop. You can try this new tool out by hitting the F1 key in the Podman Desktop UI. [#4081](https://github.com/containers/podman-desktop/pull/4081) &amp;&amp; [#3979](https://github.com/containers/podman-desktop/pull/3979)
+
+<ReactPlayer playing controls url="https://user-images.githubusercontent.com/436777/270362431-5aaa6a1b-6df5-4b66-a811-cdd148d02ad6.mp4" width='100%' height='100%' />
+
+### Expanded "Summary" tab for Kubernetes pods
+
+Kubernetes pods now offer a more comprehensive set of information under the "Summary" tab, including networking, volumes, environment variables, and other key metadata.
+
+<ReactPlayer playing controls url="https://user-images.githubusercontent.com/6422176/272972815-bed96f3a-6b13-45d3-a13b-74eacb27a4cd.mov" width='100%' height='100%' />
+
+### Environment file support
+
+When creating a container from the Images list, there's now an option to provide an environment file to set env variables for the new container. [#4026](https://github.com/containers/podman-desktop/pull/4026) &amp;&amp; [#4025](https://github.com/containers/podman-desktop/pull/4025)
+
+### Enhancements to the Settings area
+
+The user experience for enabling or disabling Docker compatibility is improved, with a new entry in the Settings > Preferences screen that includes contextual guidance. [#4093](https://github.com/containers/podman-desktop/pull/4093)
+
+<ReactPlayer playing controls url="https://user-images.githubusercontent.com/6422176/270497318-902b2566-62ad-4ee6-87af-6a9a2705de99.mov" width='100%' height='100%' />
+
+Enhanced logic for displaying or hiding properties listed under the Settings > Preferences screens is now available. [#4159](https://github.com/containers/podman-desktop/pull/4159)
```

Comment:
Needs a 'why' (e.g. how we're using it), or maybe just move to the extensions API section.

Reply:
Good idea, moved to the extensions API section.